### DISCUSSION TITLE
Mounting Local Volume on CasaOS

### DISCUSSION THREAD

**Discussion by hxck** [Lissy93/dashy#1236] [Help and Support]
Installing Dashy as a custom app with the docker compose through CasaOS' import doesn't work at the moment on stable 0.4.3, but I was able to build the image through CLI and import it into CasaOS so I have a working deployment. That said, I'm not sure how to mount a local storage for the config as what I usually do doesn't seem to work. I have:

/DATA/AppData/dashy/conf.yml:/app/public/conf.yml

It looks [like this](https://xbb.hxck.art/NUtu8/SUTaJOha01.png) in the CasaOS settings. [This](https://xbb.hxck.art/NUtu8/rOMidOlI94.png) is what I see in Dozzle after starting the container. The [file exists](https://xbb.hxck.art/NUtu8/bIPEnedu12.png) and I can see it in FileBrowser. With all of that, Dashy still only shows the default Getting Started area.

Any idea what I'm doing wrong? I'm still newish to self-hosting and linux. Thanks in advance.

Debian 12 Bookworm base
Docker version 24.0.2, build cb74dfc
[CasaOS](https://casaos.io/) 0.4.3

**Comment by SantiOrteal** [comment]
I'm not sure whether you're still having problems, but I just installed Dashy with CasaOS. 

There were no issues when I used Portainer to install the stack.

**Reply by hxck** [reply]
Oh man I switched up a long time ago, but thanks.
